lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1799,8 +1799,11 @@ getNetworkParameters
     -> Handler ApiNetworkParameters
 getNetworkParameters (_block0, np, _st) nl = do
     pp <- liftIO $ NW.getProtocolParameters nl
-    let (apiNetworkParams, epochNoM) =
-            toApiNetworkParameters np { protocolParameters = pp }
+    let pastHorizon :: PastHorizonException -> IO W.SlottingParameters
+        pastHorizon _ = pure (slottingParameters np)
```

Comment:
In the unlikely event this happens, I think we should rather do something else than using the genesis slotting parameters. Say the tip is in Shelley of the `Byron;Shelley;Allegra` mode, then the genesis slotting params would be misleading, no?

Seems unlikely, and maybe not the _most_ important, but maybe add a comment about it at least?

Reply:
I added a comment. It should never happen because we ask the node for slotting parameters of its current tip. That should always be within the horizon. The only time I have seen the PastHorizonException is when the node tip was at genesis.